src/App.tsx

```diff
@@ -74,11 +60,7 @@ export const App: React.FC = () => {
                 )}
 
                 {author && loaded && !hasError && posts.length > 0 && (
-                  <PostsList
-                    posts={posts}
-                    selectedPostId={selectedPost?.id}
-                    onPostSelected={setSelectedPost}
-                  />
+                  <PostsList />
```

Comment:
The 'PostsList' component is used correctly. Ensure that this component is properly defined and exported in './components/PostsList'.